src/components/visualizations/common/proptypes/headline.ts

```diff
@@ -0,0 +1,38 @@
+// (C) 2007-2018 GoodData Corporation
+import { Execution, AFM } from '@gooddata/typings';
+import * as PropTypes from 'prop-types';
+import { CSSProperties } from 'react';
+
+const HeadlineDataItem = PropTypes.shape({
+    localIdentifier: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+    value: PropTypes.string,
+    format: PropTypes.string,
+    isDrillable: PropTypes.bool
+});
+
+export interface IHeadlineDataItem  {
+    localIdentifier: AFM.Identifier;
+    title?: string;
+    value: Execution.DataValue;
+    format?: string;
+    isDrillable?: boolean;
+}
+
+export interface IFormattedHeadlineDataItem {
+    cssStyle: CSSProperties;
+    value: Execution.DataValue;
+    isValueEmpty: boolean;
+}
+
+export const HeadlineData = PropTypes.shape({
```

Comment:
`PropTypes` suffix as its almost the same naming as `IHeadlineData`

Reply:
I dont understand

Reply:
```
export const HeadlineDataPROPTYPES = PropTypes.shape({
    primaryItem: HeadlineDataItem.isRequired,
    secondaryItem: HeadlineDataItem,
    tertiaryItem: HeadlineDataItem
});

export interface IHeadlineData {
    primaryItem: IHeadlineDataItem;
    secondaryItem?: IHeadlineDataItem;
    tertiaryItem?: IHeadlineDataItem;
}
```

Reply:
Ok. will rename to `HeadlineDataItemPropTypes` and  `HeadlineDataPropTypes`